service/service.go

```diff
@@ -89,8 +89,15 @@ type Settings struct {
 	AsyncErrorChannel chan error
 
 	// LoggingOptions provides a way to change behavior of zap logging.
+	//
+	// These options will be appended to any options passed to BuildZapLogger.
```

Comment:
It feels like we should deprecate this in favor of `BuildZapLogger` as you did on `LoggerSettings`. Any reason not to do that?